config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+H7Qek//UK0uqK0ZIMLWt69f9Fr9OvDDc6VDK3UGXSDJst+WjMYeYUk3dLqFRFkAPkqnE/vub9E0OaIu4mnBKF4rfm/DTh4UMWiftIWXBiNzPCqI51a3e7dz4JLuM/vraNucg5akpX6cDlKyPPg/XQfZKOTG5yqXAXkCd76oZxPNSnZJFTOG3JujXKtRARLpdaNUXSHZi+adlN6EX6aZxAOMFoJd9LH8oW/Tpo9tu6tuGgFiekay0NWJbGOW2r5GOOmOuB94e/ymaANsioueTdiBx50He4EomdO6sXCWnVq0IpbaSe5PQhjeTeHBN2/vPAjsdYL6QvCddFDF13yN12JK3a3E21iw3JOmcUBtZeJCmO/3YnScw6a0draLUYntN6man/lUrkUbqQtK3ThksHj4M398uwB1U2gVm--/iKZ1yXswZ9t8XiU--+KxeFwIENpp9dvtoerRp3A==
```

Comment:
ignore this file in file '.gitignore'. Create empty file 'credentials.yml.enc.example'